tests/ecmp/test_fgnhg.py

```diff
@@ -330,3 +328,6 @@ def test_fg_ecmp(ansible_adhoc, testbed, ptfadapter, duthost, ptfhost):
     # IPv6 test
     port_list, ip_to_port, bank_0_port, bank_1_port = setup_test_config(ptfadapter, duthost, ptfhost, cfg_facts, router_mac, net_ports, DEFAULT_VLAN_IPv6, PREFIX_IPv6)
     fg_ecmp(ptfhost, duthost, router_mac, net_ports, port_list, ip_to_port, bank_0_port, bank_1_port, PREFIX_IPv6) 
+
+    # Cleanup all configuration to default after the test completes
+    cleanup(duthost)
```

Comment:
@anish-n What will happen if test fail before the clean up is done? It looks like that we will face the same issue again. Can we use fixture with teardown logic instead?

Reply:
Thanks, fixed the same